### PR TITLE
docs(website): add more FAQ question

### PR DESCRIPTION
**Pull request by shortcuts**
**Summary**

This question has been asked multiple times, so I think it make sense to add it to the FAQ

### CODE REVIEW

File: packages/website/docs/migrating-from-legacy.mdx [hunk 1]
```diff
@@ -118,6 +118,12 @@ Your [legacy config][11] will be parsed to a [Crawler config][12], please use [t
 
 Your credentials will remain available, but **once all the existing configs have been migrated, we will stop the daily crawl jobs**.
 
+### Why do I see two Algolia app in my dashboard?
```
Comment (Haroenv) [inline packages/website/docs/migrating-from-legacy.mdx:121]:
```suggestion
### Why do I see two Algolia apps in my dashboard?
```

File: packages/website/docs/migrating-from-legacy.mdx [hunk 2]
```diff
@@ -118,6 +118,12 @@ Your [legacy config][11] will be parsed to a [Crawler config][12], please use [t
...
+
+We did not remove access to the legacy DocSearch application (`BH4D9OD16A`) to give you the time to get familiar with our new infra. `BH4D9OD16A` will remain available until the migration has been completed for all the DocSearch users.
```
Comment (Haroenv) [inline packages/website/docs/migrating-from-legacy.mdx:123]:
```suggestion
We did not remove access to the legacy DocSearch application (`BH4D9OD16A`) to give you the time to get familiar with our new infrastructure. `BH4D9OD16A` will remain available until the migration has been completed for all the DocSearch users.
```

File: packages/website/docs/migrating-from-legacy.mdx (hunk 1 above)
Comment (shortcuts) [reply packages/website/docs/migrating-from-legacy.mdx]: Thanks <3

File: packages/website/docs/migrating-from-legacy.mdx (hunk 2 above)
Comment (shortcuts) [reply packages/website/docs/migrating-from-legacy.mdx]: Thanks <3

**Review by Haroenv — APPROVED** [review]

**Pull request merged**